src/cljs/main/broadfcui/page/test_login.cljs

```diff
@@ -0,0 +1,50 @@
+(ns broadfcui.page.test-login
+  (:require
+   [dmohs.react :as react]
+   [broadfcui.components.buttons :as buttons]
+   [broadfcui.components.foundation-tooltip :as tooltip]
+   [broadfcui.common.icons :as icons]
+   [broadfcui.common.input :as input]
+   [broadfcui.common.style :as style]
+   [broadfcui.nav :as nav]
+   [broadfcui.utils :as utils]
+   [broadfcui.utils.user :as user]
+   ))
+
+(react/defc Page
+  {:render
+   (fn [{:keys [this refs]}]
+    [:div {}
+      [:div {}
+        "Token:"
+        [input/TextField {:ref "force-token"
+                          :autoFocus true
+                          :type "password"
+                          :data-test-id "force-token"
```

Comment:
Will this be used by anything?

Reply:
dunno -  it doesn't hurt (one more prop in the DOM), and I added it in case LoadNinja wants to look at it for any reason.